crypto/x509/x509_vfy.c

```diff
@@ -2843,7 +2843,11 @@ static int build_chain(X509_STORE_CTX *ctx)
     int i;
 
     /* Our chain starts with a single untrusted element. */
-    OPENSSL_assert(num == 1 && ctx->num_untrusted == num);
+    if (!ossl_assert(num == 1 && ctx->num_untrusted == num))  {
```

Comment:
To me this (and others in this file) are actually adding value. I don't really want to remove them.